meta/.phpstorm.meta.php

```diff
@@ -505,6 +505,20 @@ function expectedReturnValues($functionReference, $values) {
     expectedArguments(\yaml_emit(), 1, YAML_ANY_ENCODING, YAML_UTF8_ENCODING, YAML_UTF16LE_ENCODING, YAML_UTF16BE_ENCODING);
     expectedArguments(\yaml_emit(), 2, YAML_ANY_BREAK, YAML_CR_BREAK, YAML_LN_BREAK, YAML_CRLN_BREAK);
 
+    expectedArguments(\AMQPExchange::delete(), 1, AMQP_NOPARAM, AMQP_IFUNUSED);
+    expectedArguments(\AMQPExchange::publish(), 2, AMQP_NOPARAM | AMQP_MANDATORY | AMQP_IMMEDIATE);
+    expectedArguments(\AMQPExchange::setFlags(), 0, AMQP_PASSIVE | AMQP_DURABLE);
+    expectedReturnValues(\AMQPExchange::getFlags(), AMQP_PASSIVE | AMQP_DURABLE);
+
+    expectedArguments(\AMQPQueue::ack(), 1, AMQP_NOPARAM, AMQP_MULTIPLE);
+    expectedArguments(\AMQPQueue::consume(), 1, AMQP_NOPARAM | AMQP_AUTOACK | AMQP_JUST_CONSUME);
+    expectedArguments(\AMQPQueue::delete(), 0, AMQP_NOPARAM, AMQP_IFUNUSED);
+    expectedArguments(\AMQPQueue::get(), 0, AMQP_NOPARAM | AMQP_AUTOACK);
```

Comment:
Looks `AMQP_NOPARAM, AMQP_AUTOACK` for me, for the only usage is `(AMQP_AUTOACK & flags) ? 1 : 0`